dts/bindings/spi/st,stm32-spi.yaml

```diff
@@ -5,11 +5,13 @@ description: STM32 SPI controller
 
 compatible: "st,stm32-spi"
 
-include: spi-controller.yaml
+include: [spi-controller.yaml, dma.yaml]
```

Comment:
This is not needed as dma properties are now part of base.yaml

Reply:
removing